### DISCUSSION TITLE
posts/2021/ab4aad48/index

### DISCUSSION THREAD

**Discussion by giscus** [GZTimeWalker/blog-comments#27] [Announcements]
# posts/2021/ab4aad48/index

关于折腾 oh-my-posh 的那些事情……安装、换主题，又因为用 kali 十分上头，给 PowerShell 写了一个看起来仿佛 kali 的主题，这篇文章便是我折腾的记录。

https://blog.gzti.me/posts/2021/ab4aad48/index.html

**Comment by Selamatpag1** [comment]
太牛啦，为啥我用了博主的主题，但是并没有像博主的一样显示python环境

**Reply by GZTimeWalker** [reply]
新版的 oh-my-posh 中不会在 home 目录显示 conda 环境（我不知道为什么要这样设计）

**Comment by Selamatpag1** [comment]
感谢回复！我们没有在home目录里，但是他也不会显示conda环境，我怀疑跟我的conda有关系，而不是oh-my-posh。
因为我今天发现我切换不了环境了QAQ。


| |
hedavid2000
|
|
***@***.***
|
---- Replied Message ----
| From | ***@***.***> |
| Date | 12/27/2024 16:13 |
| To | ***@***.***> |
| Cc | Dawei ***@***.***>,
***@***.***> |
| Subject | Re: [GZTimeWalker/blog-comments] posts/2021/ab4aad48/index (Discussion #27) |

新版的 oh-my-posh 中不会在 home 目录显示 conda 环境（我不知道为什么要这样设计）
